test/SerilogWeb.Classic.Mvc.Test/Web.config

```diff
@@ -12,14 +12,14 @@
 
     <add key="serilog:minimum-level" value="Debug" />
     <add key="serilog:using:Trace" value="Serilog.Sinks.Trace" />
-    <add key="serilog:write-to:Trace"/>
-    <add key="serilog:write-to:Trace.outputTemplate" value="{Timestamp:yyyy-MM-dd HH:mm:ss.fff zzz} [{Level}] {Message}{NewLine}{Exception} {Properties:j}"/>
+    <add key="serilog:write-to:Trace" />
+    <add key="serilog:write-to:Trace.outputTemplate" value="{Timestamp:yyyy-MM-dd HH:mm:ss.fff zzz} [{Level}] {Message}{NewLine}{Exception} {Properties:j}" />
     
-    <add key="serilog:using:SerilogWeb.Classic.Mvc" value="SerilogWeb.Classic.Mvc"/>
-    <add key="serilog:enrich:WithMvcActionName.propertyName" value="MvcActionNameTest"/>
-    <add key="serilog:enrich:WithMvcControllerName.propertyName" value="MvcControllerNameTest"/>
-    <add key="serilog:enrich:WithMvcRouteData.propertyName" value="MvcRouteDataTest"/>
-    <add key="serilog:enrich:WithMvcRouteTemplate.propertyName" value="MvcRouteTemplateTest"/>
+    <add key="serilog:using:SerilogWeb.Classic.Mvc" value="SerilogWeb.Classic.Mvc" />
+    <add key="serilog:enrich:WithMvcActionName.propertyName" value="MvcActionNameTest" />
+    <add key="serilog:enrich:WithMvcControllerName.propertyName" value="MvcControllerNameTest" />
+    <add key="serilog:enrich:WithMvcRouteData.propertyName" value="MvcRouteDataTest" />
+    <add key="serilog:enrich:WithMvcRouteTemplate.propertyName" value="MvcRouteTemplateTest" />
```

Comment:
It looks like there are only white space changes here ... could you possibly revert those so we have a smaller diff ?

Reply:
Yes, very annoying. This tends to happen when updating some NuGet packages. Do you know of any remedy against that side-effect?

Reply:
Appart from double-checking every time we commit, I don't know a solution 🤷‍♂  .... or just commit this file with the spaces once and for all, but well ...